modules/ui/sidebar.js

```diff
@@ -54,6 +54,12 @@ export function uiSidebar(context) {
             .on(_pointerPrefix + 'down.sidebar-resizer', pointerdown);
 
         var downPointerId, lastClientX, containerLocGetter;
+        var flagCollapse = false;
+        var flagExpand = false;
+
+        var holdWindowSize = 0;
```

Comment:
Perhaps a different name would make the logic clearer:

```suggestion
        var previousWindowSize = 0;
```